rapid7_insightvm/plugin.spec.yaml

```diff
@@ -1655,6 +1655,11 @@ actions:
         description: ID of the site to scan
         type: string
         required: true
+      hosts:
+        title: Hosts
+        description: The hosts that should be included in the scan
+        type: "[]string"
```

Comment:
@PhilippBehmer Can you add `example` keys for this Scan action while you're in here? and regen / create the raw example input section in help.md for it. The latest version of icon-plugin will autogenerate the JSON section from the example keys in the spec.

e..g.

Example input:
```
{
  "id": "blah",
   ....
}
```